FHIR-us-davinci-deqm.xml

```diff
@@ -13,6 +13,7 @@
 <artifactPageExtension value="-definitions"/>
 <artifactPageExtension value="-examples"/>
 <artifactPageExtension value="-mappings"/>
+<artifact id="StructureDefinition/module-definition-library-cqfm" key="StructureDefinition-module-definition-library-cqfm" name="CQFM Module Definition Library"/>
```

Comment:
Why was this added?

Reply:
I got no clue.  Wasn't anything I intentionally added.  But I do have a tendency to rely on the `_genonce.sh` tool to re-build the IG, and I'm fairly sure that I ran `_updatePublisher.sh` sometime in the past month.  Maybe an update to dependencies caused it to get compiled or fetched from elsewhere?

Reply:
File removed.